client/asset/eth/eth.go

```diff
@@ -4816,13 +4817,21 @@ func (getGas) EstimateFunding(ctx context.Context, net dex.Network, assetID, con
 	return nil
 }
 
-// ReturnETH returns the estimation wallet's Ethereum balance to a specified
+// Return returns the estimation wallet's Ethereum balance to a specified
```

Comment:
Still says "Ethereum balance" but it seems you're now returning whatever, ETH or MATIC, or tokens now.